app/utils/Formatters.scala

```diff
@@ -58,4 +59,18 @@ object Formatters {
   def buildUppercaseQueryParams(name: String, value: Option[String]): String =
     value match { case Some(paramValue) if paramValue != "" => s"&$name=${paramValue.toUpperCase.trim}"; case _ => "" }
 
+  def formatCurrency[A: Numeric](total: A): String = {
+    val currencyFormatter = getCurrencyInstance(java.util.Locale.UK)
+    currencyFormatter.setRoundingMode(java.math.RoundingMode.DOWN)
+
+    currencyFormatter.format(total)
+  }
+
+  def formatCurrencyRoundedToPounds[A: Numeric](total: A): String = {
```

Comment:
could we add a `maximumFractionDigits:Option[Int] = None` parameter to `formatCurrency` and use it like so:

```scala
maximumFractionDigits.foreach(digits => currencyFormatter.setMaximumFractionDigits(digits))
```

then the `formatCurrencyRoundedToPounds` method would pass-through to it, like so `formatCurrency(total, Some(0))`

Reply:
I like it, updating now